Stripe/STPAPIRequest.h

```diff
@@ -20,10 +20,22 @@ typedef void(^STPAPIResponseBlock)(ResponseType object, NSHTTPURLResponse *respo
                                  serializer:(ResponseType)serializer
                                  completion:(STPAPIResponseBlock)completion;
 
++ (NSURLSessionDataTask *)postWithAPIClient:(STPAPIClient *)apiClient
+                                    endpoint:(NSString *)endpoint
+                                  parameters:(NSDictionary *)parameters
+                                 serializers:(NSArray<ResponseType>*)serializers
```

Comment:
should we go ahead and rename `serializers` to `deserializers` (or in a different PR)?